sqlstore/migrations/0001_block_signers.sql

```diff
@@ -2,7 +2,7 @@
 
 CREATE extension IF NOT EXISTS timescaledb;
 
-CREATE SCHEMA metrics;
+CREATE SCHEMA IF NOT EXISTS metrics;
```

Comment:
Maybe remove this line? Along with the `DROP SCHEMA IF EXISTS metrics;`?
reasoning: It is no longer controlled by goose but by other mechanisms.

Reply:
Ahh yess we can do it :)